pkg/datapath/loader/netlink.go

```diff
@@ -391,7 +391,7 @@ func setupVethPair(name, peerName string) error {
 // the first step of datapath initialization, then performs the setup (and
 // creation, if needed) of those interfaces. It returns two links and an error.
 // By default, it sets up the veth pair - cilium_host and cilium_net.
-func SetupBaseDevice(mtu int) (netlink.Link, netlink.Link, error) {
+func (l *loader) SetupBaseDevice(mtu int) (netlink.Link, netlink.Link, error) {
```

Comment:
We can avoid leaving this exported and adding this to the loader API at all by making `createDevices()` in `daemon_privileged_tests.go` create a dummy device named `defaults.HostDevice`. All other uses are internal, and for the sake of the privileged tests, any device will do. cilium_net isn't even touched.